Neos.ContentRepository.Core/Classes/Feature/ContentStreamCreation/Event/ContentStreamWasCreated.php

```diff
@@ -28,23 +27,18 @@ final class ContentStreamWasCreated implements EventInterface
 {
     public function __construct(
         public readonly ContentStreamId $contentStreamId,
-        public readonly UserId $initiatingUserId
     ) {
     }
 
     public static function fromArray(array $values): self
     {
         return new self(
             ContentStreamId::fromString($values['contentStreamId']),
-            UserId::fromString($values['initiatingUserId']),
         );
     }
 
     public function jsonSerialize(): array
     {
-        return [
-            'contentStreamId' => $this->contentStreamId,
-            'initiatingUserId' => $this->initiatingUserId,
-        ];
+        return get_object_vars($this);
```

Comment:
hm, can we keep this explicit? :) I feel this is too dangerous to have it implicit (same for the other changes)

Reply:
As discussed, `get_object_vars()` is not really less explicit, it just makes all props of the event part of their payload which makes sense for events IMO and I think the risk of a typo is greater with the repetition